cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -465,29 +469,74 @@ func (s *Server) Index(args *indexArgs) (state indexState, err error) {
 		return indexStateEmpty, createEmptyShard(args)
 	}
 
+	repositoryName := args.BuildOptions().RepositoryDescription.Name
```

Comment:
can you more directly access the name? Calling buildoptions seems like wasted work.